nbs/core.ipynb

```diff
@@ -339,6 +339,8 @@
     "        self.id_col = id_col\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #33.    try:
</pre><p>We use fastcore's test here, so please reformat this as:</p><pre class="ql-syntax" spellcheck="false">test_fail(lambda: nf.fit(test_df2), contains=expected_error_msg)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/894/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>